src/main/kotlin/com/mparticle/kits/RoktKit.kt

```diff
@@ -222,6 +222,8 @@ class RoktKit :
             }
         }
 
+        finalAttributes.putAll(filterAttributes(attributes, configuration))
```

Comment:
On web, we start by merging the [placement attributes onto the user attributes](https://github.com/mparticle-integrations/mparticle-javascript-integration-rokt/blob/main/src/Rokt-Kit.js#L215-L216) to create a new placementAttributes variable.  Then later on we do a [filter on that using the user attribute filters](https://github.com/mparticle-integrations/mparticle-javascript-integration-rokt/blob/main/src/Rokt-Kit.js#L237-L241).  This is to ensure an attribute that is passed to `selectPlacements` will also be filtered out if a customer chooses to in the MP UI.

Reply:
The scenario I'm thinking of is as follows:

1. User has attributes of `faveColor` and `faveNumber`, and developer wants to filter `faveColor` in our UI.
2. If the developer passes `faveColor` and `faveNumber` in `selectPlacements`, these will be part of `finalAttributes` and not filtered out, right?